npm/design-system/src/stories/util.ts

```diff
@@ -8,7 +8,7 @@ export const createStorybookConfig = (config: Meta): Meta => config
 /**
  * Compact way of declaring a new story
  */
-export const createStory = <T>(template: Story<T>, args?: Partial<T>) => {
+export const createStory = <T = any>(template: Story<T>, args?: Partial<T>): Story<T> => {
```

Comment:
Why is the default of `any` necessary? Can't you just omit that?

Reply:
No matter what I try, I can't seem to get this working :(

Reply:
Ah, I figured out why it's being weird. I disabled TypeScript on the specs in the `tsconfig`, which means TS wasn't trying very hard to properly show the types. Removing the `exclude` clause has everything work correctly without these type changes (although you might want to change the default value of `T` to `{}`:

```
createStory = <T = {}>(...) => {...}
```